packages/app/src/app/pages/Dashboard/Content/routes/Settings/TeamSettings/ManageSubscription/paddle.tsx

```diff
@@ -11,42 +11,70 @@ import {
   Icon,
   Tooltip,
 } from '@codesandbox/components';
+import { useWorkspaceSubscription } from 'app/hooks/useWorkspaceSubscription';
+import track from '@codesandbox/common/lib/utils/analytics';
 
 export const Paddle = () => {
-  const { activeTeamInfo: team } = useAppState();
+  const { subscription } = useWorkspaceSubscription();
   const actions = useActions();
 
-  return (
-    <Stack direction="vertical" gap={2}>
-      {!team.subscription.cancelAt ? (
-        <Tooltip
-          label={`Next invoice of ${team.subscription.currency} ${(
-            (team.subscription.quantity * team.subscription.unitPrice) /
-            100
-          ).toFixed(2)} (excl. tax) scheduled for ${format(
-            new Date(team.subscription.nextBillDate),
-            'PP'
-          )}`}
+  if (subscription.cancelAt) {
+    return (
+      <Stack
+        css={{
+          height: '100%',
+        }}
+        direction="vertical"
+        justify="space-between"
+      >
+        <Text size={3} css={{ color: '#F7CC66' }}>
+          Your access to Pro features will expire on on{' '}
+          {format(new Date(subscription.cancelAt), 'PP')}. After this period,
+          your team will be automatically migrated to the Free plan.
+        </Text>
+        <Button
+          autoWidth
+          variant="link"
+          css={css({
+            color: '#F7CC66',
+            padding: 0,
+            fontSize: 3,
+          })}
+          onClick={() => actions.pro.reactivateWorkspaceSubscription()}
         >
-          <Stack align="center" gap={1}>
-            <Text size={3} variant="muted">
-              Next invoice: {team.subscription.currency}{' '}
-              {(
-                (team.subscription.quantity * team.subscription.unitPrice) /
-                100
-              ).toFixed(2)}{' '}
-            </Text>
-            <Text variant="muted">
-              <Icon name="info" size={12} />
-            </Text>
-          </Stack>
-        </Tooltip>
-      ) : null}
+          Upgrade to pro
+        </Button>
+      </Stack>
+    );
+  }
 
+  return (
+    <Stack direction="vertical" gap={2}>
+      <Tooltip
+        label={`Next invoice of ${subscription.currency} ${(
+          (subscription.quantity * subscription.unitPrice) /
+          100
+        ).toFixed(2)} (excl. tax) scheduled for ${format(
+          new Date(subscription.nextBillDate),
+          'PP'
```

Comment:
Nitpick - we could move the calculations to separate `const`s.

Reply:
Removed inline calculations here: [74939d3](https://github.com/codesandbox/codesandbox-client/pull/7195/commits/74939d32f835c52a6388873e81a47ae8603e0f9c).